gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Sat Aug 20 15:56:50 PDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-bin.zip
```

Comment:
No. Don't try sneaking this crap in. This breaks peoples setups.

Reply:
I'll quote my comment:

> Updating Gradle is necessary for the newest Shadow plugin to work. The newest version compatible with Gradle 2.14, Shadow 1.2.4, clashes with the ASM in ForgeGradle itself, as noted in #542.

Reply:
Yes, I saw that, it shouldn't be an issue tho. With the proper non-packaging of ASM.